src/SFA.DAS.ApprenticeCommitments/DTOs/RevisionDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.ApprenticeCommitments.DTOs
+{
+    public class RevisionDto
```

Comment:
The name implies it's a RevisionDto and it's not

Reply:
I did think about calling it TimelineDto - its actually a revision - I prefer to stick to the actual table name it resembles so staying with that.

Reply:
We have an entity in our Database called Revision, if I was a new member to the team and saw RevisionDTO I would assume it directly relates to this entity. It is clearly not a revision, but the difference between two revisions. As a good citizen we should unambiguous about what the DTO represents

Reply:
The object that gets returns is an apprenticeshipDto - not homepage dto or whatever its used for.. yes the revisions are being returned for a timeline display in this context but could be used on the frontend for whatever.